types/matrix-js-sdk/index.d.ts

```diff
@@ -270,7 +270,61 @@ export interface LoginPayload {
    */
   well_known?: any;
 }
-
+export interface IncludeEventContext {
+  before_limit?: number;
+  after_limit?: number;
+  include_profile?: boolean;
+}
+export interface EventContext {
+  start: string;
+  end: string;
+  profile_info?: {
+    [userId: string]: {
+      displayname:	string;
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/matrix-js-sdk/index.d.ts;
  Line: 283;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

Reply:
Ok, I'll fix it